cassandra/cqlengine/connection.py

```diff
@@ -28,67 +28,200 @@
 
 NOT_SET = _NOT_SET  # required for passing timeout to Session.execute
 
-Host = namedtuple('Host', ['name', 'port'])
-
-cluster = None
```

Comment:
I like the object oriented approach, but wondering if we should set these module attributes when the default connection is made, so we don't lose attributes that were not explicitly "private".